doc/sources/index.rst

```diff
@@ -49,38 +49,79 @@ Important Links
 - `About daal4py <https://github.com/intel/scikit-learn-intelex/tree/master/daal4py>`_
 
 .. toctree::
-   :caption: About
+   :caption: Getting Started
    :hidden:
+   :maxdepth: 2
 
-   acceleration.rst
+   gsg.rst 
+   quick-start.rst
    what-is-patching.rst
-   Medium Blogs <blogs.rst>
+   
+.. toctree::
+   :caption: Features
+   :hidden:
+   :maxdepth: 2
+
+   algorithms.rst
+   oneAPI and GPU support <oneapi-gpu.rst>
+   distributed-mode.rst
+   verbose.rst
+
+.. toctree::
+   :caption: Releases
+   :hidden:
+   :maxdepth: 2
+
+   releases.rst 
+
+.. toctree::
+   :caption: Installation Guide
+   :hidden:
+   :maxdepth: 2
+
    System Requirements <system-requirements.rst>
    memory-requirements.rst
+   installation.rst  
```

Comment:
I think the expectation of Installation Guide is to go fo installation link first of all and not go via system requirements.

Also i don't think that Memory requirements fall under Installation guide block - those are runtime considerations on using our version

Reply:
prefer that users understand system requirements before starting installation

Reply:
put installation at first in https://github.com/intel/scikit-learn-intelex/pull/1323/commits/b66d55aded070c3285a1d12c33bd0e0266dd19db